### ISSUE TITLE
Request to Implement Chapter Thumbnails for Komga in Aidoku

### ISSUE THREAD

**Issue by Jamerrone** [Aidoku/Aidoku#444]
I have a suggestion/request regarding the integration of Komga with Aidoku. Currently, most readers that support Komga are able to display a thumbnail for each chapter, utilizing a built-in feature of Komga. This functionality would be particularly beneficial for users who store entire volumes in Komga rather than individual chapters.

Would it be possible to implement this feature in Aidoku as well? I believe it would enhance the user experience, especially for those of us managing our collections in this manner.

I understand this might be a niche request, but I would greatly appreciate it if you could consider adding this functionality.

Thank you for your time and consideration.

**Comment by Skittyblock** [comment]
yeah chapter thumbnail support is planned for the next iteration of the source api (hopefully landing in 0.7). I can't guarantee immediate ui support, but it should hopefully follow soon after if not in the first 0.7 build.

**Comment by Jamerrone** [comment]
I did not expect this to be a planned feature or even something you were willing to consider. What a pleasant surprise!
